spaces/domains.json

```
@@ -877,6 +877,7 @@
   "dao.bglnode.online": "bgldao.eth",
   "vote.primapes.gg": "primapes.eth",
   "governance.encode.org": "encodeorg.eth",
+  "vote.hodl.finance": "hodl.finance",
```

You will need to edit your ENS here
```suggestion
  "vote.hodl.finance": "hft.eth",
```